free-1.md

```diff
@@ -56,13 +155,33 @@ spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
+I am not entirely sure how to implement languages quite yet, so I don't think I
+have an accurate judgment of this at the moment. I do, however, think that this
+may require a lot of systems code to work in the context of an editor. At this
+point I would still think language design would be a major part of the project,
+so my guess would be 50-70% of time would be spent on language design. 
 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
-
+I think this is a pretty ambitious project. While I think the impact is relatively
+small, and adds some convenience to a very specific group of people, it may be
+difficult to implement. Furthermore, if I am able to somehow mix pure text or 
+markup and efficiently create graphs in place (unlikely, this will more likely
```

Comment:
This also reminds my of IPython (now Jupyter) notebooks - they let you intermix LaTeX, Markdown, Python, etc and allows for inline graphs using matplotlib, for example. I wonder if maybe extending the Jupyter notebook concept and providing another language they can use (maybe backed by matplotlib) that simplifies creating graphs.